proto-lens/src/Data/ProtoLens/TextFormat.hs

```diff
@@ -52,28 +59,38 @@ import qualified Data.ProtoLens.TextFormat.Parser as Parser
 
 -- | Pretty-print the given message into a human-readable form.
 pprintMessage :: Message msg => msg -> Doc
-pprintMessage = pprintMessage' descriptor
+pprintMessage = pprintMessageWithRegistry mempty
+
+-- | Pretty-print the given message into human-readable form, using the given
+-- 'Registry' to decode @google.protobuf.Any@ values.
```

Comment:
s/decode/encode

Reply:
Looking back at the old reviews, you told me to change encode to decode and decode to encode :) Can we just leave it? 

In reality, it's doing both. I could say 'to encode Any values into the text format'

Reply:
Sure, sorry about that.  Whichever one makes more sense to you.